includes/class-amp-theme-support.php

```diff
@@ -143,7 +135,7 @@ class AMP_Theme_Support {
 	 * @since 0.7
 	 */
 	public static function init() {
-		self::$init_start_time = microtime( true );
+		do_action( 'amp_server_timing_start', 'init', 'AMP Output Buffer' );
```

Comment:
Let's add phpdoc for all actions. We can label them `@internal`.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/5005/commits/5dc57b27a159c1f08f36ee2e34f11b47f4c17a71